plugins/modules/edpm_os_net_config.py

```diff
@@ -207,7 +209,7 @@ def main():
     detailed_exit_codes = args['detailed_exit_codes']
     safe_defaults = args['safe_defaults']
     use_nmsate = args['use_nmstate']
-    return_codes = [0]
+    return_codes = [0,1]
```

Comment:
i think no need add 1 here. only 0 is for success to check failed or not.

Reply:
Have removed this file now